packages/filebrowser/src/listing.ts

```diff
@@ -1622,8 +1622,8 @@ export class DirListing extends Widget {
   }
 
   protected translator: ITranslator;
+  protected _model: FilterFileBrowserModel;
```

Comment:
I am making this one protected without renaming it, because there is already a getter defined but I would like to override this getter with something equivalent to:

```typescript
get model(): MyCustomModel {
  return this._model;
}
```